packages/cli/src/tools/releaseChecker/printNewRelease.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ */
+import chalk from 'chalk';
+import logger from '../logger';
+import type {Release} from './getLatestRelease';
+import cacheManager from './releaseCacheManager';
+
+/**
+ * Notifies the user that a newer version of React Native is available.
+ */
+export default function(latestRelease: Release, currentVersion: string) {
```

Comment:
```suggestion
export default function printNewRelease(latestRelease: Release, currentVersion: string) {
```
this will likely need the `eslint --fix` run as well